specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/mfe.json

```diff
@@ -0,0 +1,1719 @@
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        }
+      }
+    },
+    "DatastoreContents": {
+      "required": [
+        "type"
+      ],
+      "type": "object",
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/ContentsType"
+        },
+        "azureDataLake": {
+          "$ref": "#/definitions/AzureDataLakeSection"
+        },
+        "azureMySql": {
+          "$ref": "#/definitions/AzureMySqlSection"
+        },
+        "azurePostgreSql": {
+          "$ref": "#/definitions/AzurePostgreSqlSection"
+        },
+        "azureSqlDatabase": {
+          "$ref": "#/definitions/AzureSqlDatabaseSection"
+        },
+        "azureStorage": {
+          "$ref": "#/definitions/AzureStorageSection"
+        },
+        "glusterFs": {
+          "$ref": "#/definitions/GlusterFsSection"
+        }
+      }
+    },
+    "OriginType": {
+      "enum": [
+        "Synapse"
+      ],
+      "type": "string"
+    },
+    "LinkedInfo": {
+      "type": "object",
+      "properties": {
+        "linkedId": {
+          "type": "string"
+        },
+        "linkedResourceName": {
+          "type": "string"
+        },
+        "origin": {
+          "$ref": "#/definitions/OriginType"
+        }
+      }
+    },
+    "DatastoreProperties": {
+      "required": [
+        "contents"
+      ],
+      "type": "object",
+      "properties": {
+        "contents": {
+          "$ref": "#/definitions/DatastoreContents"
+        },
+        "description": {
+          "type": "string"
+        },
+        "hasBeenValidated": {
+          "type": "boolean",
+          "readOnly": true
+        },
+        "id": {
+          "format": "uuid",
+          "type": "string",
+          "readOnly": true
+        },
+        "isDefault": {
+          "type": "boolean",
+          "readOnly": true
+        },
+        "linkedInfo": {
+          "$ref": "#/definitions/LinkedInfo"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "DatastorePropertiesResource": {
+      "description": "Azure Resource Manager resource Envelope",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DatastoreProperties"
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "DatastorePropertiesResourcePaginatedResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DatastorePropertiesResource"
+          }
+        },
+        "continuationToken": {
```

Comment:
continuationToken is not part of the paginated response format.  nextLink and value are the only accepted properties